RevenueCatUI/Views/FooterView.swift

```diff
@@ -101,20 +105,28 @@ private struct LinkButton: View {
     var body: some View {
         let bundle = Localization.localizedBundle(self.locale)
 
-        ViewThatFits {
-            ForEach(self.titles, id: \.self) { title in
-                Link(
-                    bundle.localizedString(
-                        forKey: title,
-                        value: nil,
-                        table: nil
-                    ),
-                    destination: self.url
-                )
+        if #available(iOS 16.0, macOS 13.0, tvOS 16.0, *) {
+            ViewThatFits {
+                ForEach(self.titles, id: \.self) { title in
+                    self.link(for: title, bundle: bundle)
+                }
             }
+        } else if let first = self.titles.first {
+            self.link(for: first, bundle: bundle)
```

Comment:
Why is this only showing one link? 🤔

Reply:
It's the same as above, we can't use `ViewThatFits`.